packages/orbit-components/src/utils/Slide/index.js

```diff
@@ -16,14 +16,16 @@ const getMaxHeight = ({ maxHeight }) => {
 export const StyledSlide = styled.div`
   position: relative;
   width: 100%;
-  transition: ${transition(["max-height"], "fast", "linear")};
+  transition: ${({ transitionDuration }) =>
+    transition(["max-height"], transitionDuration, "linear")};
   max-height: ${getMaxHeight};
   overflow: ${({ transitionFinished }) => !transitionFinished && "hidden"};
   visibility: ${({ visible }) => !visible && "hidden"};
 `;
 
 StyledSlide.defaultProps = {
   theme: defaultTheme,
+  transitionDuration: "fast",
```

Comment:
`Slide` needs to have this default prop defined, not `StyledSlide` 😄 `StyledSlide` will get this indirectly from `Slide`.